src/installer.js

```diff
@@ -22,6 +22,10 @@ var getPixmapPath = function (options) {
   return path.join('/share/pixmaps', options.id + '.png')
 }
 
+var sanitizePackageNameParts = function (parts) {
+  return parts.map(part => part.replace(/[^a-z0-9]+/gi, '_'))
```

Comment:
I think I'd prefer this without the `+` so that consecutive invalid characters get turned into one consecutive `_` each, instead of collapsing into only one `_`.